articles/overview/how-to-learn-quantum-computing.md

```diff
@@ -0,0 +1,46 @@
+---
+title: How to learn quantum computing
+description: 
+author: natke
+ms.author:  nakersha
+ms.date: 10/16/2019
+ms.topic: article
+uid: microsoft.quantum.overview.learn
+---
+
+# How to learn quantum computing
+
+Learn how to write your first quantum program.
+
+## Getting the basics: what do you need to know?
+
+Fortunately, you don’t need to know quantum physics to start writing quantum applications. However, there are some concepts that are especially important to understand quantum computing. But don’t worry if you don’t know something! We will give you the resources to learn the necessary tools in a light and interactive way.
+
+* Linear algebra (vector and matrices): quantum states are vectors in a vector space over the complex numbers. Quantum gates are just linear operations acting on these states. Here you can find a tutorial to get everything you need to know about linear algebra. (Link to a Binder for the tutorial on linear algebra).
```

Comment:
It needs a lot of improvement, to be sure, but would it be helpful to link to the existing linear algebra concepts page (https://docs.microsoft.com/quantum/concepts/vectors-and-matrices)?